specs/models/Step.schema.json

```diff
@@ -132,6 +132,56 @@
     },
     "name": {
       "type": "string"
+    },
+    "properties": {
+      "type": "object",
+      "$ref": "./MappingProperties.schema.json"
+    },
+    "expressionContext": {
```

Comment:
I don't think we need this - i.e. I don't think we have a requirement yet for a user to define a top-level context.

Reply:
If you consider an entity model 'User' with one property “name” of type "string". If we want to map it to the source file, 
```
{
	"names": ["A", "B", "C", "D"]
}
```
so as to generate 4 different entity instances corresponding to each of the names, we would want "expressionContext" set to "/names" so that we get a mapping like this and 4 entity instances can be created. Currently, we set the "expressionContext" to "/" always at top level

```
<m:mapping xmlns:map="http://marklogic.com/xdmp/map" xmlns:m="http://marklogic.com/entity-services/mapping">
  <m:entity name="User">
    <User xmlns="" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
          <m:optional><name xsi:type="xs:integer"><m:val>.</m:val></name></m:optional>
    </User>
  </m:entity>
  <m:output>
    <m:for-each>
      <m:select>/names</m:select>
        <m:call-template name="User"/>
    </m:for-each>
  </m:output>
</m:mapping>
```

Reply:
I believe that requirement is tangential to the "Map related entities" feature; that requirement is really part of a "Map multiple entities of the target entity type" feature which I don't believe is part of 5.5 yet. It could be; but it's not part of "Map related entities", so I don't think we should try to support it yet.